disnake/embeds.py

```diff
@@ -762,3 +769,17 @@ def to_dict(self) -> EmbedData:
             result["title"] = self.title
 
         return result  # type: ignore
+
+    @classmethod
+    def set_default_colour(cls, value: Union[int, Colour]):
```

Comment:
The contents of this method and `colour.setter` above should be refactored into a single method to avoid duplicating code

Reply:
Talked about on discord, not possible or recommended.